public/media_engagement/imaqal_media_engagement.js

```diff
@@ -201,13 +204,19 @@ const update = (data) => {
     let colorReceived = d3.scaleOrdinal(d3.schemeCategory10).domain(receivedKeys)
     let colorSent = d3.scaleOrdinal(d3.schemeCategory10).domain(sentKeys)
 
-
     // set scale domains
     y_total_failed_sms.domain([0, d3.max(data, function (d) { return d.total_errored; })]);
 
-    drawOneDayReceivedGraph()
-    drawOneDaySentGraph()
-    
+    if (view == "1day") {
+        drawOneDayReceivedGraph()
+        drawOneDaySentGraph()
+    }
+
+    else if (view == "10min") {
+        draw10MinReceivedGraph()
+        draw10MinSentGraph()
+    }
```

Comment:
If it's something else what should happen? Warning in console? Warning on the UI?